clang/test/CodeGen/pointer-overflow.c

```diff
@@ -10,3 +10,24 @@ void test(void) {
   // DEFAULT: getelementptr inbounds nuw i32, ptr
   // FWRAPV-POINTER: getelementptr i32, ptr
 }
+
+struct S {
+  int a;
+  int b;
+  int c: 10;
```

Comment:
I am curious why did you add bit-fields in specifically to this test? Are they particularly different wrt other types of fields like arrays, pointers, member pointers etc that we would expect them to break?